src/hotspot/share/opto/phasetype.hpp

```diff
@@ -157,11 +156,19 @@ class PhaseNameIter {
 
 class PhaseNameValidator {
  private:
+  CHeapBitMap _mask;
   bool _valid;
+  bool _set;
   char* _bad;
 
+
  public:
-  PhaseNameValidator(ccstrlist option, uint64_t& mask) : _valid(true), _bad(nullptr) {
+  PhaseNameValidator(ccstrlist option)
+                    : _mask(PHASE_NUM_TYPES, mtCompiler),
+                      _valid(true),
+                      _set(false),
+                      _bad(nullptr)
```

Comment:
I think the colon belongs up on the same line as `PhaseNameValidator(ccstrlist option)`, and the initializations only should be indented by 2 whitespaces.

Reply:
Alright, sounds good to me (now changed). The previous style is the same as for `Compile::Compile` in `compile.cpp`, but the style differs between different parts of HotSpot.